src/ProjectList/ProjectDetails.tsx

```diff
@@ -1,5 +1,5 @@
 import { List } from "@raycast/api";
-import { Project } from "./defaultProjects";
+import { Project } from "../defaultProjects";
```

Comment:
Moved `defaultProjects` to the `src` root folder because it's used by both commands.